internal/security/bootstrapper/command/setupacl/aclroles_test.go

```diff
@@ -17,27 +17,29 @@ package setupacl
 
 import (
 	"context"
+	"errors"
```

Comment:
should move this test to go-mod-secret?

Reply:
Agree.  This should have gone along with aclroles.go.